src/module/actor/SR5Actor.ts

```diff
@@ -199,178 +218,212 @@ export class SR5Actor extends Actor {
+     * @param data
+     */
+    static prepareModifiers(data: SR5ActorData) {
+        if (!data.modifiers) data.modifiers = {};
+        const modifiers = {};
+        let miscTabModifiers = [
+            'soak',
+            'drain',
+            'armor',
+            'physical_limit',
+            'social_limit',
+            'mental_limit',
+            'stun_track',
+            'physical_track',
+            'meat_initiative',
+            'meat_initiative_dice',
+            'astral_initiative',
+            'astral_initiative_dice',
+            'matrix_initiative',
+            'matrix_initiative_dice',
+            'composure',
+            'lift_carry',
+            'judge_intentions',
+            'memory',
+            'walk',
+            'run',
+            'defense',
+            'wound_tolerance',
+            'essence',
+            'fade',
+        ];
+        miscTabModifiers.sort();
+        // force global to the top
+        miscTabModifiers.unshift('global');
+
+        for (let item of miscTabModifiers) {
+            modifiers[item] = Number(data.modifiers[item]) || 0;
+        }
+
+        data.modifiers = modifiers;
+    }
 
-        // INITIATIVE
-        const init = data.initiative;
-        init.meatspace.base.base = attributes.intuition.value + attributes.reaction.value + modifiers['meat_initiative'];
-        init.meatspace.dice.base = 1 + modifiers['meat_initiative_dice'];
-        init.astral.base.base = attributes.intuition.value * 2 + modifiers['astral_initiative'];
-        init.astral.dice.base = 2 + modifiers['astral_initiative_dice'];
-        init.matrix.base.base = attributes.intuition.value + data.matrix.data_processing.value + modifiers['matrix_initiative'];
-        init.matrix.dice.base = data.matrix.hot_sim ? 4 : 3 + modifiers['matrix_initiative_dice'];
-        if (init.perception === 'matrix') init.current = init.matrix;
-        else if (init.perception === 'astral') init.current = init.astral;
+    /**
+     * Prepare actor data for initiative
+     * @param data
+     */
+    static prepareInitiative(data: SR5ActorData) {
+        const { initiative, attributes, modifiers, matrix } = data;
+        initiative.meatspace.base.base = attributes.intuition.value + attributes.reaction.value + Number(modifiers['meat_initiative']);
+        initiative.meatspace.dice.base = 1 + Number(modifiers['meat_initiative_dice']);
+        initiative.astral.base.base = attributes.intuition.value * 2 + Number(modifiers['astral_initiative']);
+        initiative.astral.dice.base = 2 + Number(modifiers['astral_initiative_dice']);
+        initiative.matrix.base.base = attributes.intuition.value + data.matrix.data_processing.value + Number(modifiers['matrix_initiative']);
+        initiative.matrix.dice.base = matrix.hot_sim ? 4 : 3 + Number(modifiers['matrix_initiative_dice']);
+        if (initiative.perception === 'matrix') initiative.current = initiative.matrix;
```

Comment:
Use curly braces for legibility
```suggestion
        if (initiative.perception === 'matrix') {
            initiative.current = initiative.matrix;
        }
```